<!-- 
RSS generated by JIRA (9.12.27#9120027-sha1:edc4490121e366e9e7bd2213d532dbe7e028fc5d) at Tue Sep 30 18:19:46 UTC 2025

It is possible to restrict the fields that are returned in this document by specifying the 'field' parameter in your request.
For example, to request only the issue key and summary append 'field=key&field=summary' to the URL of your request.
-->
<rss version="0.92" >
<channel>
    <title>Java Bug System</title>
    <link>https://bugs.openjdk.org</link>
    <description>This file is an XML representation of an issue</description>
    <language>en-us</language>    <build-info>
        <version>9.12.27</version>
        <build-number>9120027</build-number>
        <build-date>02-09-2025</build-date>
    </build-info>


<item>
            <title>[JDK-8305600] java/lang/invoke/lambda/LogGeneratedClassesTest.java fails after JDK-8304846 and JDK-8202110</title>
                <link>https://bugs.openjdk.org/browse/JDK-8305600</link>
                <project id="10100" key="JDK">JDK</project>
                    <description>The following test failed in the JDK21 CI:&lt;br/&gt;
&lt;br/&gt;
java/lang/invoke/lambda/LogGeneratedClassesTest.java&lt;br/&gt;
&lt;br/&gt;
after the integration of the following fix:&lt;br/&gt;
&lt;br/&gt;
&lt;a href=&quot;https://bugs.openjdk.org/browse/JDK-8304846&quot; title=&quot;Provide a shared utility to dump generated classes defined via Lookup API&quot; class=&quot;issue-link&quot; data-issue-key=&quot;JDK-8304846&quot;&gt;&lt;strike&gt;JDK-8304846&lt;/strike&gt;&lt;/a&gt; Provide a shared utility to dump generated classes defined via Lookup API&lt;br/&gt;
&lt;br/&gt;
It looks like &lt;a href=&quot;https://bugs.openjdk.org/browse/JDK-8304846&quot; title=&quot;Provide a shared utility to dump generated classes defined via Lookup API&quot; class=&quot;issue-link&quot; data-issue-key=&quot;JDK-8304846&quot;&gt;&lt;strike&gt;JDK-8304846&lt;/strike&gt;&lt;/a&gt; modified the existing test.&lt;br/&gt;
&lt;br/&gt;
Here&amp;#39;s a snippet from the log file:&lt;br/&gt;
&lt;br/&gt;
----------System.out:(189/12466)----------&lt;br/&gt;
config LogGeneratedClassesTest.setup(): success&lt;br/&gt;
test LogGeneratedClassesTest.testDumpDirIsFile(): failure&lt;br/&gt;
java.lang.AssertionError: only show error once expected [1] but found [0]&lt;br/&gt;
	at org.testng.Assert.fail(Assert.java:99)&lt;br/&gt;
	at org.testng.Assert.failNotEquals(Assert.java:1037)&lt;br/&gt;
	at org.testng.Assert.assertEqualsImpl(Assert.java:140)&lt;br/&gt;
	at org.testng.Assert.assertEquals(Assert.java:122)&lt;br/&gt;
	at org.testng.Assert.assertEquals(Assert.java:797)&lt;br/&gt;
	at LogGeneratedClassesTest.testDumpDirIsFile(LogGeneratedClassesTest.java:177)&lt;br/&gt;
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)&lt;br/&gt;
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)&lt;br/&gt;
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)&lt;br/&gt;
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)&lt;br/&gt;
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)&lt;br/&gt;
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)&lt;br/&gt;
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)&lt;br/&gt;
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)&lt;br/&gt;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)&lt;br/&gt;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)&lt;br/&gt;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&lt;br/&gt;
	at org.testng.TestRunner.privateRun(TestRunner.java:764)&lt;br/&gt;
	at org.testng.TestRunner.run(TestRunner.java:585)&lt;br/&gt;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)&lt;br/&gt;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)&lt;br/&gt;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)&lt;br/&gt;
	at org.testng.SuiteRunner.run(SuiteRunner.java:286)&lt;br/&gt;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)&lt;br/&gt;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)&lt;br/&gt;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)&lt;br/&gt;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)&lt;br/&gt;
	at org.testng.TestNG.runSuites(TestNG.java:1069)&lt;br/&gt;
	at org.testng.TestNG.run(TestNG.java:1037)&lt;br/&gt;
	at com.sun.javatest.regtest.agent.TestNGRunner.main(TestNGRunner.java:93)&lt;br/&gt;
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)&lt;br/&gt;
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)&lt;br/&gt;
	at com.sun.javatest.regtest.agent.MainActionHelper$AgentVMRunnable.run(MainActionHelper.java:312)&lt;br/&gt;
	at java.base/java.lang.Thread.run(Thread.java:1623)&lt;br/&gt;
test LogGeneratedClassesTest.testDumpDirNotExist(): failure&lt;br/&gt;
java.lang.AssertionError: Two lambda captured expected [2] but found [0]&lt;br/&gt;
	at org.testng.Assert.fail(Assert.java:99)&lt;br/&gt;
	at org.testng.Assert.failNotEquals(Assert.java:1037)&lt;br/&gt;
	at org.testng.Assert.assertEqualsImpl(Assert.java:140)&lt;br/&gt;
	at org.testng.Assert.assertEquals(Assert.java:122)&lt;br/&gt;
	at org.testng.Assert.assertEquals(Assert.java:797)&lt;br/&gt;
	at LogGeneratedClassesTest.testDumpDirNotExist(LogGeneratedClassesTest.java:155)&lt;br/&gt;
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)&lt;br/&gt;
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)&lt;br/&gt;
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)&lt;br/&gt;
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)&lt;br/&gt;
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)&lt;br/&gt;
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)&lt;br/&gt;
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)&lt;br/&gt;
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)&lt;br/&gt;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)&lt;br/&gt;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)&lt;br/&gt;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&lt;br/&gt;
	at org.testng.TestRunner.privateRun(TestRunner.java:764)&lt;br/&gt;
	at org.testng.TestRunner.run(TestRunner.java:585)&lt;br/&gt;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)&lt;br/&gt;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)&lt;br/&gt;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)&lt;br/&gt;
	at org.testng.SuiteRunner.run(SuiteRunner.java:286)&lt;br/&gt;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)&lt;br/&gt;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)&lt;br/&gt;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)&lt;br/&gt;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)&lt;br/&gt;
	at org.testng.TestNG.runSuites(TestNG.java:1069)&lt;br/&gt;
	at org.testng.TestNG.run(TestNG.java:1037)&lt;br/&gt;
	at com.sun.javatest.regtest.agent.TestNGRunner.main(TestNGRunner.java:93)&lt;br/&gt;
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)&lt;br/&gt;
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)&lt;br/&gt;
	at com.sun.javatest.regtest.agent.MainActionHelper$AgentVMRunnable.run(MainActionHelper.java:312)&lt;br/&gt;
	at java.base/java.lang.Thread.run(Thread.java:1623)&lt;br/&gt;
test LogGeneratedClassesTest.testDumpDirNotWritable(): failure&lt;br/&gt;
java.lang.AssertionError: only show error once expected [1] but found [0]&lt;br/&gt;
	at org.testng.Assert.fail(Assert.java:99)&lt;br/&gt;
	at org.testng.Assert.failNotEquals(Assert.java:1037)&lt;br/&gt;
	at org.testng.Assert.assertEqualsImpl(Assert.java:140)&lt;br/&gt;
	at org.testng.Assert.assertEquals(Assert.java:122)&lt;br/&gt;
	at org.testng.Assert.assertEquals(Assert.java:797)&lt;br/&gt;
	at LogGeneratedClassesTest.testDumpDirNotWritable(LogGeneratedClassesTest.java:239)&lt;br/&gt;
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)&lt;br/&gt;
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)&lt;br/&gt;
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)&lt;br/&gt;
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)&lt;br/&gt;
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)&lt;br/&gt;
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)&lt;br/&gt;
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)&lt;br/&gt;
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)&lt;br/&gt;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)&lt;br/&gt;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)&lt;br/&gt;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&lt;br/&gt;
	at org.testng.TestRunner.privateRun(TestRunner.java:764)&lt;br/&gt;
	at org.testng.TestRunner.run(TestRunner.java:585)&lt;br/&gt;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)&lt;br/&gt;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)&lt;br/&gt;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)&lt;br/&gt;
	at org.testng.SuiteRunner.run(SuiteRunner.java:286)&lt;br/&gt;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)&lt;br/&gt;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)&lt;br/&gt;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)&lt;br/&gt;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)&lt;br/&gt;
	at org.testng.TestNG.runSuites(TestNG.java:1069)&lt;br/&gt;
	at org.testng.TestNG.run(TestNG.java:1037)&lt;br/&gt;
	at com.sun.javatest.regtest.agent.TestNGRunner.main(TestNGRunner.java:93)&lt;br/&gt;
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)&lt;br/&gt;
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)&lt;br/&gt;
	at com.sun.javatest.regtest.agent.MainActionHelper$AgentVMRunnable.run(MainActionHelper.java:312)&lt;br/&gt;
	at java.base/java.lang.Thread.run(Thread.java:1623)&lt;br/&gt;
test LogGeneratedClassesTest.testLogging(): failure&lt;br/&gt;
java.lang.AssertionError: Two lambda captured expected [2] but found [0]&lt;br/&gt;
	at org.testng.Assert.fail(Assert.java:99)&lt;br/&gt;
	at org.testng.Assert.failNotEquals(Assert.java:1037)&lt;br/&gt;
	at org.testng.Assert.assertEqualsImpl(Assert.java:140)&lt;br/&gt;
	at org.testng.Assert.assertEquals(Assert.java:122)&lt;br/&gt;
	at org.testng.Assert.assertEquals(Assert.java:797)&lt;br/&gt;
	at LogGeneratedClassesTest.testLogging(LogGeneratedClassesTest.java:130)&lt;br/&gt;
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)&lt;br/&gt;
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)&lt;br/&gt;
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)&lt;br/&gt;
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)&lt;br/&gt;
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)&lt;br/&gt;
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)&lt;br/&gt;
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)&lt;br/&gt;
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)&lt;br/&gt;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)&lt;br/&gt;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)&lt;br/&gt;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&lt;br/&gt;
	at org.testng.TestRunner.privateRun(TestRunner.java:764)&lt;br/&gt;
	at org.testng.TestRunner.run(TestRunner.java:585)&lt;br/&gt;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)&lt;br/&gt;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)&lt;br/&gt;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)&lt;br/&gt;
	at org.testng.SuiteRunner.run(SuiteRunner.java:286)&lt;br/&gt;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)&lt;br/&gt;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)&lt;br/&gt;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)&lt;br/&gt;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)&lt;br/&gt;
	at org.testng.TestNG.runSuites(TestNG.java:1069)&lt;br/&gt;
	at org.testng.TestNG.run(TestNG.java:1037)&lt;br/&gt;
	at com.sun.javatest.regtest.agent.TestNGRunner.main(TestNGRunner.java:93)&lt;br/&gt;
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)&lt;br/&gt;
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)&lt;br/&gt;
	at com.sun.javatest.regtest.agent.MainActionHelper$AgentVMRunnable.run(MainActionHelper.java:312)&lt;br/&gt;
	at java.base/java.lang.Thread.run(Thread.java:1623)&lt;br/&gt;
test LogGeneratedClassesTest.testLoggingException(): failure&lt;br/&gt;
java.lang.AssertionError: show error each capture expected [2] but found [0]&lt;br/&gt;
	at org.testng.Assert.fail(Assert.java:99)&lt;br/&gt;
	at org.testng.Assert.failNotEquals(Assert.java:1037)&lt;br/&gt;
	at org.testng.Assert.assertEqualsImpl(Assert.java:140)&lt;br/&gt;
	at org.testng.Assert.assertEquals(Assert.java:122)&lt;br/&gt;
	at org.testng.Assert.assertEquals(Assert.java:797)&lt;br/&gt;
	at LogGeneratedClassesTest.testLoggingException(LogGeneratedClassesTest.java:261)&lt;br/&gt;
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)&lt;br/&gt;
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)&lt;br/&gt;
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)&lt;br/&gt;
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)&lt;br/&gt;
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)&lt;br/&gt;
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)&lt;br/&gt;
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)&lt;br/&gt;
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)&lt;br/&gt;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)&lt;br/&gt;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)&lt;br/&gt;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&lt;br/&gt;
	at org.testng.TestRunner.privateRun(TestRunner.java:764)&lt;br/&gt;
	at org.testng.TestRunner.run(TestRunner.java:585)&lt;br/&gt;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)&lt;br/&gt;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)&lt;br/&gt;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)&lt;br/&gt;
	at org.testng.SuiteRunner.run(SuiteRunner.java:286)&lt;br/&gt;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)&lt;br/&gt;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)&lt;br/&gt;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)&lt;br/&gt;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)&lt;br/&gt;
	at org.testng.TestNG.runSuites(TestNG.java:1069)&lt;br/&gt;
	at org.testng.TestNG.run(TestNG.java:1037)&lt;br/&gt;
	at com.sun.javatest.regtest.agent.TestNGRunner.main(TestNGRunner.java:93)&lt;br/&gt;
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)&lt;br/&gt;
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)&lt;br/&gt;
	at com.sun.javatest.regtest.agent.MainActionHelper$AgentVMRunnable.run(MainActionHelper.java:312)&lt;br/&gt;
	at java.base/java.lang.Thread.run(Thread.java:1623)&lt;br/&gt;
test LogGeneratedClassesTest.testNotLogging(): success&lt;br/&gt;
config LogGeneratedClassesTest.cleanup(): success&lt;br/&gt;
&lt;br/&gt;
===============================================&lt;br/&gt;
java/lang/invoke/lambda/LogGeneratedClassesTest.java&lt;br/&gt;
Total tests run: 6, Passes: 1, Failures: 5, Skips: 0&lt;br/&gt;
===============================================&lt;br/&gt;
&lt;br/&gt;
----------System.err:(25/1508)----------&lt;br/&gt;
TestLambda.java:7: warning: [removal] SecurityManager in java.lang has been deprecated and marked for removal&lt;br/&gt;
&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;System.setSecurityManager(new SecurityManager());&lt;br/&gt;
&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;^&lt;br/&gt;
TestLambda.java:7: warning: [removal] setSecurityManager(SecurityManager) in System has been deprecated and marked for removal&lt;br/&gt;
&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;System.setSecurityManager(new SecurityManager());&lt;br/&gt;
&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;^&lt;br/&gt;
2 warnings&lt;br/&gt;
LongPackageName.java:7: warning: [removal] SecurityManager in java.lang has been deprecated and marked for removal&lt;br/&gt;
&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;System.setSecurityManager(new SecurityManager());&lt;br/&gt;
&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;^&lt;br/&gt;
LongPackageName.java:7: warning: [removal] setSecurityManager(SecurityManager) in System has been deprecated and marked for removal&lt;br/&gt;
&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;System.setSecurityManager(new SecurityManager());&lt;br/&gt;
&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;^&lt;br/&gt;
2 warnings&lt;br/&gt;
Unable to visit NotExist/DUMP_LAMBDA_PROXY_CLASS_FILES: java.nio.file.NoSuchFileException: NotExist/DUMP_LAMBDA_PROXY_CLASS_FILES&lt;br/&gt;
java.lang.Exception: failures: 5&lt;br/&gt;
	at com.sun.javatest.regtest.agent.TestNGRunner.main(TestNGRunner.java:95)&lt;br/&gt;
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)&lt;br/&gt;
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)&lt;br/&gt;
	at com.sun.javatest.regtest.agent.MainActionHelper$AgentVMRunnable.run(MainActionHelper.java:312)&lt;br/&gt;
	at java.base/java.lang.Thread.run(Thread.java:1623)&lt;br/&gt;
&lt;br/&gt;
JavaTest Message: Test threw exception: java.lang.Exception&lt;br/&gt;
JavaTest Message: shutting down test&lt;br/&gt;
&lt;br/&gt;
result: Failed. Execution failed: `main&amp;#39; threw exception: java.lang.Exception: failures: 5</description>
                <environment></environment>
        <key id="5097909">JDK-8305600</key>
            <summary>java/lang/invoke/lambda/LogGeneratedClassesTest.java fails after JDK-8304846 and JDK-8202110</summary>
                <type id="1" iconUrl="https://bugs.openjdk.org/secure/viewavatar?size=xsmall&amp;avatarId=14703&amp;avatarType=issuetype">Bug</type>
                                            <priority id="2" iconUrl="https://bugs.openjdk.org/images/jbsImages/p2.png">P2</priority>
                        <status id="6" iconUrl="https://bugs.openjdk.org/images/icons/statuses/closed.png" description="The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.">Closed</status>
                    <statusCategory id="3" key="done" colorName="success"/>
                                    <resolution id="1">Fixed</resolution>
                                        <assignee username="mchung">Mandy Chung</assignee>
                                    <reporter username="dcubed">Daniel Daugherty</reporter>
                        <labels>
                            <label>noreg-self</label>
                            <label>regression</label>
                            <label>tier1</label>
                            <label>ursqe-triaged</label>
                    </labels>
                <created>Tue, 4 Apr 2023 11:36:44 -0700</created>
                <updated>Thu, 2 Jan 2025 22:37:42 -0800</updated>
                            <resolved>Wed, 5 Apr 2023 15:43:17 -0700</resolved>
                                    <version>21</version>
                                    <fixVersion>21</fixVersion>
                                    <component>core-libs</component>
                        <due></due>
                            <votes>0</votes>
                                    <watches>4</watches>
                                                                                                                                                            <comments>
                            <comment id="14572104" author="dukebot" created="Wed, 5 Apr 2023 15:43:16 -0700"  >Changeset: 31270256&lt;br/&gt;
Author:    Mandy Chung &amp;lt;&lt;a href=&apos;mailto:mchung@openjdk.org&apos;&gt;mchung@openjdk.org&lt;/a&gt;&amp;gt;&lt;br/&gt;
Date:      2023-04-05 22:40:55 +0000&lt;br/&gt;
URL:       &lt;a href=&quot;https://git.openjdk.org/jdk/commit/312702567a15d67fcf69325b9cb6345f9ea026f7&quot;&gt;https://git.openjdk.org/jdk/commit/312702567a15d67fcf69325b9cb6345f9ea026f7&lt;/a&gt;&lt;br/&gt;
</comment>
                            <comment id="14572091" author="roboduke" created="Wed, 5 Apr 2023 14:11:10 -0700"  >A pull request was submitted for review.&lt;br/&gt;
URL: &lt;a href=&quot;https://git.openjdk.org/jdk/pull/13364&quot;&gt;https://git.openjdk.org/jdk/pull/13364&lt;/a&gt;&lt;br/&gt;
Date: 2023-04-05 21:03:15 +0000</comment>
                            <comment id="14572037" author="mchung" created="Wed, 5 Apr 2023 09:22:28 -0700"  >This is caused by &lt;a href=&quot;https://bugs.openjdk.org/browse/JDK-8202110&quot; title=&quot;(fs) Remove FileSystem support for resolving against a default directory (chdir configuration)&quot; class=&quot;issue-link&quot; data-issue-key=&quot;JDK-8202110&quot;&gt;&lt;strike&gt;JDK-8202110&lt;/strike&gt;&lt;/a&gt; which causes setting -Duser.dir to a path that is different from current working directory.  One change of behavior is that Files.createDirectories returns the path successfully but the directory is actually not created.   &lt;a href=&quot;https://bugs.openjdk.org/browse/JDK-8202110&quot; title=&quot;(fs) Remove FileSystem support for resolving against a default directory (chdir configuration)&quot; class=&quot;issue-link&quot; data-issue-key=&quot;JDK-8202110&quot;&gt;&lt;strike&gt;JDK-8202110&lt;/strike&gt;&lt;/a&gt;  is being backout (&lt;a href=&quot;https://bugs.openjdk.org/browse/JDK-8305664&quot; title=&quot;[BACKOUT] (fs) Remove FileSystem support for resolving against a default directory (chdir configuration)&quot; class=&quot;issue-link&quot; data-issue-key=&quot;JDK-8305664&quot;&gt;&lt;strike&gt;JDK-8305664&lt;/strike&gt;&lt;/a&gt;).</comment>
                            <comment id="14571838" author="mchung" created="Tue, 4 Apr 2023 15:36:53 -0700"  >If -Duser.dir=&amp;lt;dir&amp;gt; is set, it runs into bootstrapping issue that fails to create the directories of the file to be written; hence StackOverflowError.   It&amp;#39;s uncertain why Files.createDirectories doesn&amp;#39;t fail but continue but fails in Files::write when the bytes are written.    If -Duser.dir is not set, it will pass.    This could be related to &lt;a href=&quot;https://bugs.openjdk.org/browse/JDK-8202110&quot; title=&quot;(fs) Remove FileSystem support for resolving against a default directory (chdir configuration)&quot; class=&quot;issue-link&quot; data-issue-key=&quot;JDK-8202110&quot;&gt;&lt;strike&gt;JDK-8202110&lt;/strike&gt;&lt;/a&gt; which was not included in an earlier test run.&lt;br/&gt;
&lt;br/&gt;
Exception in thread &amp;quot;main&amp;quot; java.lang.InternalError: java.lang.InternalError: DUMP_LAMBDA_PROXY_CLASS_FILES/java/lang not exist&lt;br/&gt;
	at java.base/java.lang.invoke.InnerClassLambdaMetafactory.generateInnerClass(InnerClassLambdaMetafactory.java:365)&lt;br/&gt;
	at java.base/java.lang.invoke.InnerClassLambdaMetafactory.spinInnerClass(InnerClassLambdaMetafactory.java:287)&lt;br/&gt;
	at java.base/java.lang.invoke.InnerClassLambdaMetafactory.buildCallSite(InnerClassLambdaMetafactory.java:222)&lt;br/&gt;
	at java.base/java.lang.invoke.LambdaMetafactory.metafactory(LambdaMetafactory.java:341)&lt;br/&gt;
	at java.base/java.lang.invoke.BootstrapMethodInvoker.invoke(BootstrapMethodInvoker.java:134)&lt;br/&gt;
	at java.base/java.lang.invoke.CallSite.makeSite(CallSite.java:316)&lt;br/&gt;
	at java.base/java.lang.invoke.MethodHandleNatives.linkCallSiteImpl(MethodHandleNatives.java:274)&lt;br/&gt;
	at java.base/java.lang.invoke.MethodHandleNatives.linkCallSite(MethodHandleNatives.java:264)&lt;br/&gt;
	at java.base/java.lang.SecurityManager.addNonExportedPackages(SecurityManager.java:1232)&lt;br/&gt;
	at java.base/java.lang.SecurityManager.&amp;lt;clinit&amp;gt;(SecurityManager.java:1223)&lt;br/&gt;
	at com.example.TestLambda.main(TestLambda.java:7)&lt;br/&gt;
Caused by: java.lang.InternalError: DUMP_LAMBDA_PROXY_CLASS_FILES/java/lang not exist&lt;br/&gt;
	at java.base/jdk.internal.util.ClassFileDumper$1.run(ClassFileDumper.java:177)&lt;br/&gt;
	at java.base/jdk.internal.util.ClassFileDumper$1.run(ClassFileDumper.java:171)&lt;br/&gt;
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:319)&lt;br/&gt;
	at java.base/jdk.internal.util.ClassFileDumper.write(ClassFileDumper.java:171)&lt;br/&gt;
	at java.base/jdk.internal.util.ClassFileDumper.dumpClass(ClassFileDumper.java:155)&lt;br/&gt;
	at java.base/java.lang.invoke.MethodHandles$Lookup$ClassDefiner.defineClass(MethodHandles.java:2514)&lt;br/&gt;
	at java.base/java.lang.invoke.InnerClassLambdaMetafactory.generateInnerClass(InnerClassLambdaMetafactory.java:362)&lt;br/&gt;
	... 10 more</comment>
                            <comment id="14571789" author="dcubed" created="Tue, 4 Apr 2023 11:38:00 -0700"  >[~mchung] - Please take a look at these Tier1 failures.</comment>
                    </comments>
                <issuelinks>
                            <issuelinktype id="10000">
                    <name>Blocks</name>
                                                                <inwardlinks description="is blocked by">
                                        <issuelink>
            <issuekey id="5097979">JDK-8305664</issuekey>
        </issuelink>
                            </inwardlinks>
                                    </issuelinktype>
                            <issuelinktype id="10003">
                    <name>Relates</name>
                                                                <inwardlinks description="relates to">
                                        <issuelink>
            <issuekey id="5097029">JDK-8304846</issuekey>
        </issuelink>
                            </inwardlinks>
                                    </issuelinktype>
                    </issuelinks>
                <attachments>
                    </attachments>
                <subtasks>
                                              <subtask id="5097911">JDK-8305602</subtask>
                                    </subtasks>
                <customfields>
                                                                            <customfield id="customfield_11700" key="com.atlassian.jira.plugins.jira-development-integration-plugin:devsummary">
                        <customfieldname>Development</customfieldname>
                        <customfieldvalues>
                            
                        </customfieldvalues>
                    </customfield>
                                                                                                                        <customfield id="customfield_10600" key="com.oracle.jira.javabugsystem-jira-plugin:jbs-fixedBackportedCustomfield">
                        <customfieldname>Fixed</customfieldname>
                        <customfieldvalues>
                            
                        </customfieldvalues>
                    </customfield>
                                                                                                                                                                                                                                                                                                                                                                                    <customfield id="customfield_11100" key="com.pyxis.greenhopper.jira:gh-lexo-rank">
                        <customfieldname>Rank</customfieldname>
                        <customfieldvalues>
                            <customfieldvalue>0|i310n7:</customfieldvalue>

                        </customfieldvalues>
                    </customfield>
                                                                <customfield id="customfield_11004" key="com.pyxis.greenhopper.jira:gh-global-rank">
                        <customfieldname>Rank (Obsolete)</customfieldname>
                        <customfieldvalues>
                            <customfieldvalue>9223372036854775807</customfieldvalue>
                        </customfieldvalues>
                    </customfield>
                                                                                            <customfield id="customfield_10006" key="com.atlassian.jira.plugin.system.customfieldtypes:select">
                        <customfieldname>Resolved In Build</customfieldname>
                        <customfieldvalues>
                                <customfieldvalue key="17354"><![CDATA[b18]]></customfieldvalue>

                        </customfieldvalues>
                    </customfield>
                                                                                            <customfield id="customfield_10008" key="com.oracle.jira.jira-subcomponent-plugin:oracle-subComponentField">
                        <customfieldname>Subcomponent</customfieldname>
                        <customfieldvalues>
                             <customfieldvalue key="213"><![CDATA[java.lang.invoke]]></customfieldvalue> 
                        </customfieldvalues>
                    </customfield>
                                                                                                                        <customfield id="customfield_10601" key="com.oracle.jira.javabugsystem-jira-plugin:jbs-targetBackportedCustomfield">
                        <customfieldname>Targeted</customfieldname>
                        <customfieldvalues>
                            
                        </customfieldvalues>
                    </customfield>
                                                                                                                        <customfield id="customfield_10100" key="com.atlassian.jira.plugin.system.customfieldtypes:radiobuttons">
                        <customfieldname>Verification</customfieldname>
                        <customfieldvalues>
                                <customfieldvalue key="17000"><![CDATA[Verified]]></customfieldvalue>

                        </customfieldvalues>
                    </customfield>
                                                                </customfields>
    </item>
</channel>
</rss>